sandboxes/bubble/index.js

```diff
@@ -0,0 +1,6 @@
+import Vue from 'vue'
+import App from '../src/App.vue'
```

Comment:
```suggestion
import App from './src/App.vue'
```

Maybe there should be that path?

Reply:
sure. fix